frontend/model/notifications/selectors.js

```diff
@@ -39,17 +39,53 @@ sbp('sbp/selectors/register', {
       timestamp: data.createdDate ? new Date(data.createdDate).getTime() : Date.now(),
       type
     }
+    const rootState = sbp('chelonia/rootState')
+    if (!rootState.notifications) {
+      rootState.notifications = { items: [], status: {} }
+    }
+    if (rootState.notifications.items.some(item => item.hash === notification.hash)) {
+      // We cannot throw here, as this code might be called from within a contract side effect.
+      return console.error('This notification is already in the store.')
+    }
+    const index = rootState.notifications.items.findLastIndex(item => item < notification.timestamp)
```

Comment:
`item` cannot be compared to a timestamp like this

Reply:
Good catch